### PR TITLE
ui-core: add button size type

### PR DESCRIPTION
**Pull request by ghost**
Resolves #[84](https://github.com/OpenRailAssociation/osrd-ui/issues/84).

### CODE REVIEW

**Review by Yohh — APPROVED** [review]
nice! thanks for your contribution!

**Review by SharglutDev — APPROVED** [review]

File: ui-core/src/components/buttons/Button.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import React, { ReactNode } from 'react';
 import { Gear } from '@osrd-project/ui-icons';
 
 type ButtonVariant = 'Normal' | 'Cancel' | 'Quiet' | 'Destructive' | 'Primary';
+type ButtonSize = 'large' | 'medium' | 'small'
```
Comment (emersion) [inline ui-core/src/components/buttons/Button.tsx]: Nit: missing `;` at end of line here

**Comment by emersion** [comment]
Minor detail about commit message title: we generally use the "front:" prefix for anything that touches the frontend.

(Also it seems like there is a small typo: "component"!)

**Comment by clarani** [comment]
Actually in this repo (osrd-ui), you can prefix your commit by "ui-core" ;)
"front:" is for the main repo

**Comment by ghost** [comment]
Thank you, everyone, for the feedback. I think I made the appropriate edits :)

**Review by emersion — APPROVED** [review]
Yup, LGTM :)

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark: Thanks for the PR ! :)

**Pull request merged**